### DISCUSSION TITLE
Reduce required repair count of Machine Maniac talent

### DISCUSSION THREAD

**Discussion by BiffW** [FakeFishGames/Barotrauma#14760] [Ideas and suggestions]
Through my experiences, it usually takes much more time to fully unlock this talent than an Engineer fully unlock Junction Junkie. As an early-game talent, it's bonus should not be delayed for so long. I suggest required count should be reduced by 40%~60%.

**Comment by bookslayer10** [comment]
Can't you still just repeatedly damage and repair a door with a plasma cutter and wrench to cheese Machine Maniac?

That should be fixed ASAP, then the talent can be balanced properly.

**Reply by BiffW** [reply]
I can also overload the grid and repeatedly damage and repair junction boxes to progress Junction Junkie. Shouldn't that be fixed too?

**Comment by Regalis11** [comment]
This has been addressed in our private repo now, changes coming up in a future update.
